lib/components-schema-v1_0_x.ts

```diff
@@ -293,6 +293,28 @@ export const componentsDefinitionSchema_v1_0_x = {
                                     },
                                 },
                             },
+                            {
+                                additionalProperties: false,
+                                required: ['type', 'defaultConfig', 'viewLink'],
+                                properties: {
+                                    type: {
+                                        enum: ['interactive'],
+                                        description: 'Configuration of interactive directive',
+                                    },
+                                    defaultConfig: {
+                                        type: 'object',
+                                        description: 'Default configuration of interactive directive',
+                                    },
+                                    editLink: {
```

Comment:
Looks like it's possible to automatically validate urls:
https://spacetelescope.github.io/understanding-json-schema/reference/string.html#built-in-formats

Reply:
yeah

Reply:
not sure how strict it should be, I think general 'uri' should be enough

Reply:
Think so too.